src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -284,6 +309,43 @@ static bool ShouldBeFilteredOutOfExpandedCompletionList(VSCompletionItem item, I
                 // 2. or, all associated expanders are unselected, therefore should be excluded
                 return associatedWithUnselectedExpander;
             }
+
+            static ImmutableArray<CompletionItemWithHighlight> GetHighlightedList(
```

Comment:
honestly, if the perf is fine, then CompletionItem should just expose the Matched spans on it directly (instead of needing these wrapper structs).

Reply:
hmm? What do you mean "CompletionItem should just expose the Matched spans"?

Reply:
oh... yeah, brain fart (i'm a bit sick today).  that makes no sense at all.  bleagh, i'm sorry.

Reply:
np, it's comforting to know this could happen to even the best of us :)